src/lib/profiles/data-management/Current/EventLoggingTags.h

```diff
@@ -195,6 +195,30 @@ enum
     kTag_Args  = 2  ///< An array of arguments to be sent along with the token message.
 };
 
+/**
+ * @brief
+ *  Tags for persisting events
+ */
+enum
+{
+    kTag_SerializedEvent                         = 1,
+    kTag_PersistEvent_EventBuffer,
```

Comment:
Always give specific values to tag enums.  Otherwise you run the risk of someone inserting a value, causing all the other values to change, and breaking compatibility with previously encoded data.